foundations/06-cascade-fix/solution/solution.html

```diff
@@ -9,15 +9,15 @@
     <link rel="stylesheet" href="solution.css">
   </head>
   <body>
-    <p class="para">I'm just a paragraph with red text!</p>
-    <p class="para small-para">I'm a smaller paragraph, also with red text!</p>
+    <p class="para">I'm just a paragraph with bold text!</p>
+    <p class="para small-para">I'm a smaller paragraph, also with bold text!</p>
 
     <button id="confirm-button" class="button confirm">Confirm</button>
     <button id="cancel-button" class="button cancel">Cancel</button>
 
-    <div class="text">I'm a div with green text!</div>
-    <div class="text">I'm a div with green text and a child div!
-      <div class="text child">I'm a smaller child div with red text.</div>
+    <div class="text">I'm a div with thin text!</div>
+    <div class="text">I'm a div with thin text and a child div!
+      <div class="text child">I'm a smaller child div with bold text.</div>
```

Comment:
```suggestion
      <div class="text child">I'm a smaller child div with extra bold text.</div>
```
**suggestion:** Adding the "extra" to the wording in the `solution.html`. You can commit these right from the comments if you'd like!